dynatrace-oneagent-operator-google-marketplace/chart/dynatrace-oneagent-operator/templates/configmap.yaml

```diff
@@ -979,8 +979,26 @@ data:
       {{- end }}
 
       {{- if .Values.oneagent.resources }}
-      resources: {{- toYaml .Values.oneagent.resources | nindent 4 }}
-      {{- end }}
+      resources:
```

Comment:
Since you're missing the leading `-` - does it break the YAML perhaps?

Also, is this only for OneAgentAPM as it's now, or should also be for OneAgent? The normal Helm chart uses it only for the OneAgent object.

Reply:
Did work without the dashes but added them just in case